adapters/kagome/CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ add_executable(kagome-adapter
 
 target_link_libraries(kagome-adapter
   Boost::program_options
+  WAVM::libWAVM
```

Comment:
This should not be necessary. Otherwise that is a bug in kagome's CMake Package Config

Reply:
Seems to be working now. Removed.